exported/localized/de_patch/text/conversations/03_neketaka_vailian_district/03_bs_rdc_gang.stringtable

```diff
@@ -16,7 +16,7 @@
     </Entry>
     <Entry>
       <ID>4</ID>
-      <DefaultText>„Ich wette, Shrimp könnte dich über die Reling werfen. He, Shrimp!“</DefaultText>
+      <DefaultText>„Ich wette, Garnele könnte dich über die Reling werfen. He, Garnele!“</DefaultText>
```

Comment:
Hm,ich glaube, Shrimp wäre besser. Ist einfach von den Silben her kürzer und leichter auszusprechen. Spitznamen sind ja in der Regel auch eher eine Verkürzung und Shrimp ist auch nicht so unbekannt im deutschen Sprachgebrauch.